src/third_party/simple-beast-client/CMakeLists.txt

```diff
@@ -7,7 +7,8 @@ find_package(Boost 1.66 REQUIRED COMPONENTS
 find_package(OpenSSL)
 find_package(Threads)
 find_package(cppcodec)
-target_include_directories(simple-beast-client INTERFACE
+# SYSTEM INTERFACE means we don't emit warnings.
```

Comment:
nit: it's just system that [supresses warnings](https://cmake.org/cmake/help/latest/command/target_include_directories.html).

And is that a good idea?

Reply:
Given inifinite time/resources we'd have this as a part of the genny toolchain (precompiled) and/or we'd fix the issue in the 3p lib and upstream it. As it is, there is nothing genny maintainers are expected to do with compiler warnigns coming from simple-beast-client.